.github/workflows/flatpak.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     env:
       FLATPAK_BUILD_PATH: flatpak_app/files/share
     container:
-      image: bilelmoussaoui/flatpak-github-actions:kde-6.3
+      image: flatpak/flatpak-github-actions:kde-6.3
```

Comment:
They still didn't switch to github registry for container so it's still bilelmoussaoui for containers.

Reply:
They didn’t update there README https://github.com/flatpak/flatpak-github-actions so change is correct

Reply:
No the change is incorrect. The docker image is still not published on the flatpak docker hub or github packages (https://hub.docker.com/u/flatpak), only on https://hub.docker.com/u/bilelmoussaoui

Reply:
> They didn’t update there README https://github.com/flatpak/flatpak-github-actions so change is correct

The pull request to switch to Github registry as `flatpak` is not merge, for now it's in docker registry as `bilelmoussaoui`.